Automattic/woocommerce-payments
# TypeScript migration: Affirm and Afterpay payment details components

Closes https://github.com/Automattic/woocommerce-payments/issues/7388

#### Changes proposed in this Pull Request
This PR migrates Affirm and Afterpay/Clearpay components used on the transaction details page to TypeScript. 









#### Testing instructions





* `npm run listen` on the server to make sure the transactions list is up-to-date
* Complete a purchase with Affirm
* Navigate to `Payments -> Transactions` and click to the transaction details
* Confirm that the _**Payment method**_ section is displayed correctly
* Repeat similar test with Afterpay/Clearpay payment method

> 💡 See https://github.com/Automattic/woocommerce-payments/pull/7345's to have a better understanding on how the **_Payment method_** section should look like.

-------------------


**Post merge**
